catroid/src/main/res/layout-ar/brick_wait_until.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?><!--
```

Comment:
> what that means "Line does not match expected header line of '^\W*$'."

Just change to
```xml
<?xml version="1.0" encoding="utf-8"?>
<!--
```

More details at [pull/2189](https://github.com/Catrobat/Catroid/pull/2189).

Reply:
thanks,